src/panoptes/pocs/focuser/birger.py

```diff
@@ -47,77 +41,23 @@ class Focuser(AbstractSerialFocuser):
     Args:
         name (str, optional): default 'Birger Focuser'
         model (str, optional): default 'Canon EF-232'
-        initial_position (int, optional): if given the focuser will drive to this encoder position
-            following initialisation.
-        dev_node_pattern (str, optional): Unix shell pattern to use to identify device nodes that
-            may have a Birger adaptor attached. Default is '/dev/tty.USA49*.?', which is intended
-            to match all the nodes created by Tripplite Keyway USA-49 USB-serial adaptors, as
-            used at the time of writing by Huntsman.
-
-    Additional positonal and keyword arguments are passed to the base class, AbstractFocuser. See
-    that class' documentation for a complete list.
+        max_command_retries (int, optional): Maximum number of times to retry a command. Default 5
```

Comment:
Do we need a `max_retries` for these focusers?

Reply:
Yes we do, it is used as the maximum number of times to retry a command when it isn't properly sent